exercises/prime-factors/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "for": {
+    "description": "returns prime factors for the given input number",
```

Comment:
Ok, It's probably fine, further discussion here: https://github.com/exercism/x-common/pull/507#pullrequestreview-18471819